extensions-core/s3-extensions/src/main/java/org/apache/druid/storage/s3/ServerSideEncryptingAmazonS3.java

```diff
@@ -65,11 +68,21 @@ public static Builder builder()
 
   private final AmazonS3 amazonS3;
   private final ServerSideEncryption serverSideEncryption;
+  private final TransferManager transferManager;
 
-  public ServerSideEncryptingAmazonS3(AmazonS3 amazonS3, ServerSideEncryption serverSideEncryption)
+  public ServerSideEncryptingAmazonS3(AmazonS3 amazonS3, ServerSideEncryption serverSideEncryption, S3TransferConfig transferConfig)
```

Comment:
Add a test for the constructor change

Reply:
The unit tests have been updated for this change in the same commit: https://github.com/confluentinc/druid/pull/171/commits/a3701f22436f6c454f9c9ba07313d8e84fbd04d7#diff-532b1c66e0f6c9efdda06611bbb0a242f0436b8ddc8c68cc47aff69c616eca60